tests/server/test_handlers.py

```diff
@@ -612,12 +611,67 @@ def test_search_model_versions(mock_get_request_message, mock_model_registry_sto
             status_message=None,
         ),
     ]
-    mock_model_registry_store.search_model_versions.return_value = mvds
+    mock_get_request_message.return_value = SearchModelVersions(filter="source_path = 'A/B/CD'")
+    mock_model_registry_store.search_model_versions.return_value = PagedList(mvds, None)
     resp = _search_model_versions()
-    args, _ = mock_model_registry_store.search_model_versions.call_args
-    assert args == ("source_path = 'A/B/CD'",)
+    _, args = mock_model_registry_store.search_model_versions.call_args
+    assert args == {
+        "filter_string": "source_path = 'A/B/CD'",
+        "max_results": 100,
+        "order_by": [],
+        "page_token": "",
+    }
```

Comment:
`assert_called_once_with` or `assert_called_with` is easier to understand than extracting arguments then asserting it.

https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_once_with